README.md

```diff
@@ -7,10 +7,20 @@
 **autodE** is a Python module initially designed for the automated calculation of reaction profiles from SMILES strings of
 reactant(s) and product(s). Current features include: transition state location, conformer searching, atom mapping,
 Python wrappers for a range of electronic structure theory codes, SMILES parsing, association complex generation, and
- reaction profile generation.
+reaction profile generation.
 
+**autodE** harnesses computational practices to transform mechanistic hypotheses into conformationally sampled 3D geometries, culminating in the automated generation of full reaction profiles for intricate reactions.
+
+### Features
+
+Graph-based Approach for Reaction Analysis
+* Utilizes a graph-based approach for analyzing and mapping reaction processes, facilitating the understanding of complex chemical transformations.
+Comprehensive Reaction Profile Calculations
+* autodE offers the capability to automate the calculation of reaction energy profiles from 1D or 2D chemical representations
+   * This includes addition, substitution, elimination, migratory insertion, oxidative addition, and reductive elimination reactions.
```

Comment:
I think these points are covered briefly but also reasonably in the first paragraph. It might cleaner to add a link to the paper in the first paragraph for further details